tests/python/gaia-ui-tests/gaiatest/apps/homescreen/regions/bookmark_menu.py

```diff
@@ -20,7 +20,7 @@ class BookmarkMenu(Base):
 
     def __init__(self, marionette):
         Base.__init__(self, marionette)
-        Wait(self.marionette).until(lambda m: self.apps.displayed_app.name == self.name)
+        self.wait_to_be_displayed()
```

Comment:
Yes, here I need to define the manifest_url by:
manifest_url = '{}bookmark{}/manifest.webapp'.format(Base.DEFAULT_PROTOCOL,Base.DEFAULT_APP_HOSTNAME)